test/Microsoft.TestPlatform.AcceptanceTests/DisableAppdomainTests.cs

```diff
@@ -54,20 +54,19 @@ private void RunTests(string runnerFramework, string testAssembly, int passedTes
             { "DisableAppDomain", "true" }
         };
 
-        using var tempDir = new TempDirectory();
         var arguments = PrepareArguments(
             testAssembly,
             string.Empty,
-            GetRunsettingsFilePath(tempDir, runConfigurationDictionary),
-            FrameworkArgValue, resultsDirectory: tempDir.Path);
+            GetRunsettingsFilePath(TempDirectory, runConfigurationDictionary),
+            FrameworkArgValue, resultsDirectory: TempDirectory.Path);
 
         InvokeVsTest(arguments);
         ValidateSummaryStatus(passedTestCount, 0, 0);
     }
 
-    private string GetRunsettingsFilePath(TempDirectory tempDir, Dictionary<string, string> runConfigurationDictionary)
+    private string GetRunsettingsFilePath(TempDirectory TempDirectory, Dictionary<string, string> runConfigurationDictionary)
```

Comment:
I don't think you want this rename. Also, do we want to give the possibility to pass a temp dir or shall we use the base property directly within the method?